tap-gui/auth.hbs.md

```diff
@@ -112,6 +112,8 @@ auth:
   allowGuestAccess: true
 ```
 
+**Note** Guests cannot view catalog entities if you have permission plugin enabled.
```

Comment:
The only part that needs to stay here. With a link to the doc set-up-tap-gui-permissions-framework.hbs.md